demo/demo.md

```diff
@@ -766,3 +766,54 @@ For use cases where the field is `required`, but live validation is not wanted,
   </auro-input>
   ```
 </auro-accordion>
+
+## Swapping Values Between Inputs
+
+<div class="exampleWrapper exampleWrapper--flex">
```

Comment:
```suggestion
## Swapping Values Between Inputs

The following example illustrates how you can easily use the auro-input element combined with some additional JavaScript to easily create an experience where a user can easily swap the value from one input element to the other. 

<div class="exampleWrapper exampleWrapper--flex">
```

Reply:
Resolving without direct change due to review comments being added to a draft PR that was not ready for review.